### DISCUSSION TITLE
Is `Module.eval()` same as `Module.train(False)`?

### DISCUSSION THREAD

**Discussion by RahulBhalley** [ml-explore/mlx#538] [Q&A] [answered]
In documentation, we have `Module.eval()` and `Module.train(mode)`. Is setting mode to `False` same as using `Module.eval()`?

**Comment by angeloskath** [comment, marked as answer]
Yes it is literally how `Module.eval()` is implemented.

https://github.com/ml-explore/mlx/blob/f30e63353a3319f22be54e2315d04bc0cbbef1a7/python/mlx/nn/layers/base.py#L543

**Reply by RahulBhalley** [reply]
Ok, thanks. :)
